packages/earth-map/src/components/filter-by/FilterBy.tsx

```diff
@@ -86,6 +86,7 @@ const FilterBy = (props: IProps) => {
           <React.Fragment key={key}>
             {/* {<h2 className="ng-color-ltgray ng-text-display-s ng-margin-bottom">{key}</h2>} */}
```

Comment:
The commented h2 is meant to be displayed as a filter group title. Please use it, or remove it from code.